release-notes.txt

```diff
@@ -0,0 +1,208 @@
+BISDN Linux
+v4.6.0 Release Notes
+Date: 2022-05-03
+
+! New in this release
+- New platform Accton-AS5835-54x is now supported
+- baseboxd: bump version to 1.11.2
+- Licenses in build process were updated to use the correct SPDX identifier
+- Linux kernel updated to version 5.15.35
+- OF-DPA updated to support configuraton of MPLS pseudowires using RYU
+- FRR updated to version 8.2
+- baseboxd/OF-DPA: support added for creating and enabling Broadcom KNET interfaces
```

Comment:
This was already part of v4.5.0.

Reply:
This was 'experimental' in v4.5.0, should I reword it somehow?

Reply:
Updated the text to say '.. now fully supported'